entrypoints/teensy31.py

```diff
@@ -0,0 +1,10 @@
+import sys
+
+from kmk_keyboard_user import main
+
+if __name__ == '__main__':
+    try:
+        main()
+    except Exception as e:
+        sys.print_exception(e)
```

Comment:
This is actually the default behaviour if you just let the exception go completely unhandled, but this is still fine